pkg/model/stack.go

```diff
@@ -369,6 +370,10 @@ func validateStackName(name string) error {
 //UpdateNamespace updates the dev namespace
 func (s *Stack) UpdateNamespace(namespace string) error {
 	if namespace == "" {
+		envNamespace := os.Getenv("OKTETO_NAMESPACE")
+		if s.Namespace != "" && s.Namespace != envNamespace {
+			return fmt.Errorf("the namespace in the okteto stack manifest '%s' does not match the environment variable OKTETO_NAMESPACE '%s'", s.Namespace, namespace)
+		}
```

Comment:
should we do something like `os.SetEnv("OKTETO_NAMESPACE", s.Namespace)` at some point?

Reply:
I do't think so, we don't get `OKTETO_NAMESPACE` from other place